### ISSUE TITLE
twilio-python 6.1.0 wheel package hosted on PyPI is different from the Github official package

### ISSUE THREAD

**Issue by ye** [twilio/twilio-python#342]
**Version:** `6.1.0`
**API Subdomain (api/taskrouter/ip_messaging):** `api`

### Code Snippet
```python
>>> from twilio.exceptions import TwilioException
>>> from twilio.base.exceptions import TwilioException
>>> help(TwilioException)

>>> 
```

### Steps to Reproduce
```bash
$ mkvirtualenv -p python3 test-twilio
Running virtualenv with interpreter /usr/local/bin/python3
Using base prefix '/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6'
New python executable in /Users/ye/.envs/test-twilio/bin/python3.6
Also creating executable in /Users/ye/.envs/test-twilio/bin/python
Installing setuptools, pip, wheel...done.
virtualenvwrapper.user_scripts creating /Users/ye/.envs/test-twilio/bin/predeactivate
virtualenvwrapper.user_scripts creating /Users/ye/.envs/test-twilio/bin/postdeactivate
virtualenvwrapper.user_scripts creating /Users/ye/.envs/test-twilio/bin/preactivate
virtualenvwrapper.user_scripts creating /Users/ye/.envs/test-twilio/bin/postactivate
virtualenvwrapper.user_scripts creating /Users/ye/.envs/test-twilio/bin/get_env_details
$ pip install twilio==6.1.0
Collecting twilio==6.1.0
  Using cached twilio-6.1.0-py2.py3-none-any.whl
Collecting pyOpenSSL>=0.14 (from twilio==6.1.0)
  Using cached pyOpenSSL-17.0.0-py2.py3-none-any.whl
Collecting PyJWT>=1.4.2 (from twilio==6.1.0)
  Using cached PyJWT-1.5.0-py2.py3-none-any.whl
Requirement already satisfied: six in /Users/ye/.envs/test-twilio/lib/python3.6/site-packages (from twilio==6.1.0)
Collecting idna>=2.0.0 (from twilio==6.1.0)
  Using cached idna-2.5-py2.py3-none-any.whl
Collecting pysocks; python_version == "3.6" (from twilio==6.1.0)
  Using cached PySocks-1.6.7-py3-none-any.whl
Collecting cryptography>=1.3.4 (from twilio==6.1.0)
  Downloading cryptography-1.8.2-cp36-cp36m-macosx_10_10_x86_64.whl (1.5MB)
    100% |████████████████████████████████| 1.5MB 823kB/s 
Collecting pytz (from twilio==6.1.0)
  Using cached pytz-2017.2-py2.py3-none-any.whl
Collecting requests>=2.0.0 (from twilio==6.1.0)
  Downloading requests-2.14.2-py2.py3-none-any.whl (560kB)
    100% |████████████████████████████████| 563kB 1.9MB/s 
Collecting asn1crypto>=0.21.0 (from cryptography>=1.3.4->twilio==6.1.0)
  Using cached asn1crypto-0.22.0-py2.py3-none-any.whl
Requirement already satisfied: packaging in /Users/ye/.envs/test-twilio/lib/python3.6/site-packages (from cryptography>=1.3.4->twilio==6.1.0)
Collecting cffi>=1.4.1 (from cryptography>=1.3.4->twilio==6.1.0)
  Using cached cffi-1.10.0-cp36-cp36m-macosx_10_6_intel.whl
Requirement already satisfied: setuptools>=11.3 in /Users/ye/.envs/test-twilio/lib/python3.6/site-packages (from cryptography>=1.3.4->twilio==6.1.0)
Requirement already satisfied: pyparsing in /Users/ye/.envs/test-twilio/lib/python3.6/site-packages (from packaging->cryptography>=1.3.4->twilio==6.1.0)
Collecting pycparser (from cffi>=1.4.1->cryptography>=1.3.4->twilio==6.1.0)
Requirement already satisfied: appdirs>=1.4.0 in /Users/ye/.envs/test-twilio/lib/python3.6/site-packages (from setuptools>=11.3->cryptography>=1.3.4->twilio==6.1.0)
Installing collected packages: asn1crypto, pycparser, cffi, idna, cryptography, pyOpenSSL, PyJWT, pysocks, pytz, requests, twilio
Successfully installed PyJWT-1.5.0 asn1crypto-0.22.0 cffi-1.10.0 cryptography-1.8.2 idna-2.5 pyOpenSSL-17.0.0 pycparser-2.17 pysocks-1.6.7 pytz-2017.2 requests-2.14.2 twilio-6.1.0

$ pip show twilio
Name: twilio
Version: 6.1.0
Summary: Twilio API client and TwiML generator
Home-page: https://github.com/twilio/twilio-python/
Author: Twilio
Author-email: help@twilio.com
License: UNKNOWN
Location: /Users/ye/.envs/test-twilio/lib/python3.6/site-packages
Requires: cryptography, requests, pytz, pyOpenSSL, pysocks, idna, PyJWT, six

$ ll /Users/ye/.envs/test-twilio/lib/python3.6/site-packages/twilio/
total 112
-rw-r--r--   1 ye  staff     77 May 26 11:55 __init__.py
drwxr-xr-x  10 ye  staff    340 May 26 11:55 __pycache__
-rw-r--r--   1 ye  staff   4382 May 26 11:55 access_token.py
drwxr-xr-x  14 ye  staff    476 May 26 11:55 base
drwxr-xr-x   4 ye  staff    136 May 26 11:55 compat
-rw-r--r--   1 ye  staff    497 May 26 11:55 compat.py
drwxr-xr-x   3 ye  staff    102 May 26 11:55 conf
-rw-r--r--   1 ye  staff    111 May 26 11:55 exceptions.py
drwxr-xr-x   9 ye  staff    306 May 26 11:55 http
drwxr-xr-x   9 ye  staff    306 May 26 11:55 jwt
-rw-r--r--   1 ye  staff   1861 May 26 11:55 request_validator.py
drwxr-xr-x  23 ye  staff    782 May 26 11:55 rest
drwxr-xr-x   8 ye  staff    272 May 26 11:55 task_router
drwxr-xr-x   6 ye  staff    204 May 26 11:55 twiml
-rw-r--r--   1 ye  staff  17588 May 26 11:55 twiml.py
-rw-r--r--   1 ye  staff   5420 May 26 11:55 util.py
-rw-r--r--   1 ye  staff     76 May 26 11:55 version.py
```
As you can see above `twilio/exceptions.py` exists but in the actual the Github repository here, it has been moved into `twilio/base/.` since version 6.x. As a result, this import statement 
   
    `from twilio.exceptions import TwilioException`

should fail but it didn't. I discovered this during twilio 5.x -> 6.x migration, and it caused some bug in handling TwilioResAPIException where it should have been exposed in import but it didn't. 

Additionally, I think the migration guide (https://www.twilio.com/docs/libraries/python/migration-guide) should document the import statement changes.

IMMO, the fix is to rebuild/repackage twilio 6.1.0 wheel package clean from Git tag `6.1.0`. I understand this is an outdated version but it will help folks out there who version pinned to 6.1.0 (as they cut the version a few weeks ago and build Docker images based on it).

**Comment by efossier** [comment]
@ye Thanks for bringing this to our attention we'll address this on Tuesday.

**Comment by jmctwilio** [comment]
@ye  Hope things are well with you; it's been a while!

We've updated our build process since this report and this issue shouldn't persist.  Please let us know if it does come up again by opening another report.  Thank you and it's awesome that you're using Twilio.

**Comment by ye** [comment]
@jmctwilio thanks for addressing this in your build process to prevent this from happening again in the future. But as far as I am concerned, the issue raised in my ticket has *NOT* been fixed. the `twilio-6.1.0-py2.py3-none-any.whl` prebuilt wheel package file on pypi.python.org still is botched due to the exception.py source file on the package root directory level, whereas `twilio-6.1.0.tar.gz` source archive is correct and consistent with the Github repo. 

Now there could be issues of "repairing" older version of the wheel packages hosted on pypi due to md5 hashsum, I am unsure. But if `twilio-6.1.0-py2.py3-none-any.whl` is unfixed as-is on PyPI, any of your users who has their Dockerfile version pinned to twilio-6.1.0 will have a BAD twilio python client installed on their Docker Images. I hope I have clarified this ticket again this time.

Lastly, I will be keeping an eye out on this issue on future twilio python client library versions. Having had to spent a long time to find out the source of the issue was actually a ghost file in the upstream package is not a good way to be productive. I am sure many of your users experienced this behavior given the popularity of twilio APIs. This should be something quick to fix (if 6.1.0 version cannot be retroactively fixed, you should just remove this version of PyPI altogether and leave a note about it).
